src/utils/auth.js

```diff
@@ -0,0 +1,48 @@
+import { createKey } from '@near-js/biometric-ed25519';
```

Comment:
Is this file used anywhere? let's remove if these functions are no longer needed